client/sections.js

```diff
@@ -40,6 +40,13 @@ const sections = [
 		group: 'me',
 		secondary: true,
 	},
+	{
+		name: 'account-closed',
+		paths: [ '/me/account/closed' ],
+		module: 'me/account-close',
+		group: 'me',
+		secondary: false,
```

Comment:
Is this `secondary: false` (i.e., no sidebar) the only reason you had to create a new section?

Reply:
Yep

Reply:
You can also hide the sidebar by dispatching a `setSection( null, { hasSidebar: false } )` action in the route handler.

See how [Jetpack Connect does it](https://github.com/Automattic/wp-calypso/blob/master/client/jetpack-onboarding/controller.js#L13). That disables the sidebar for one specific route and allows you to keep a single section.

Reply:
Thanks! I've updated the PR to do that and avoid creating a new section. :)

Reply:
Nice! Today, I added a new action `hideSidebar` that should make this even easier: https://github.com/Automattic/wp-calypso/pull/37816

Reply:
Thanks! Updated to use that. :)